docs/wikidata.md

```diff
@@ -0,0 +1,92 @@
+# Wikidata Full Quickstart
+
+The following instructions take you through loading the full Wikidata Knowledge
+Base in QLever. Since Wikidata is very large at over 7 billion triples, **we
+recommend to run all of the following commands in a `tmux` or `screen` session**.
+Nevertheless we will note below when a command will take long to execute.
+
+## Download and build QLever using `docker`
+
+We build QLever using `docker` which takes care of all necessary dependencies.
+This should only take a couple of minutes
+
+    git clone --recursive https://github.com/ad-freiburg/QLever.git qlever
+    cd qlever
+    docker build -t qlever .
+
+## Download Wikidata
+
+If you already downloaded **and decrompressed** Wikidata to uncompressed Turtle
+format you can skip this step, otherwise we download and uncompress it.
+
+**Note:** This takes several hours as Wikidata is about 42 GB compressed and the
+servers are throttled. Also make sure you have enough space on the filesystem
+where you unpacked QLever or see the full
+[README](https://github.com/ad-freiburg/QLever#building-the-index) for
+instructions on using a different path for the index.
+
+**The index plus unpacked Wikidata will use up to about 2 TB.**
+
+    wget https://dumps.wikimedia.org/wikidatawiki/entities/latest-all.ttl.bz2
+    bunzip2 latest-all.ttl.bz2
+
+## Build a QLever Index
+
+Now we can build a QLever Index from the `latest-all.ttl` Wikidata Turtle file
+using the `wikidata_settings.json` file for some useful default settings for
```

Comment:
`... .json file for some ...`  this sentence is ginourmous and should probably be split into two. E.g.:
`.json file. The .json file conatains some useful...`